src/raftstore/store/fsm/apply.rs

```diff
@@ -3917,4 +3919,126 @@ mod tests {
         });
         res.unwrap_err();
     }
+
+    #[test]
+    fn test_apply_tilt() {
```

Comment:
"tilt" seems odd to me. @nrc @Hoverbear what do you think?

Reply:
me too

Reply:
I don't know what words to use. Give me a suggestion. thx.

Reply:
Hmm, maybe "backpressure" or "workload skew"?